source/FreeRTOS_ND.c

```diff
@@ -749,9 +746,6 @@
 
                     pxICMPPacket->xEthernetHeader.usFrameType = ipIPv6_FRAME_TYPE;
 
-                    configASSERT( pxEndPoint != NULL );
```

Comment:
https://github.com/FreeRTOS/FreeRTOS-Plus-TCP/blob/dev/IPv6_integration/source/FreeRTOS_ND.c#L953 
The assets are removed because the function is called only if both the checks pass. So those asserts are not possible

Reply:
I agree for the case of `prvReturnICMP_IPv6` as NULL checks are there and is an internal function, but for `FreeRTOS_SendPingRequestIPv6()` its not used anywhere in the stack and can be used by the user. Though at present the asserts seems useless it can save a lot of time in debugging (maybe in future usecases) at the expense of nothing.

Reply:
The asserts are only removed when they are not reachable. [FreeRTOS_ND.c#L953](https://github.com/FreeRTOS/FreeRTOS-Plus-TCP/blob/dev/IPv6_integration/source/FreeRTOS_ND.c#L953)

Reply:
ASSERT removed from FreeRTOS_ND as those asserts are no longer reachable because the function prvReturnICMP_IPv6 is called only when both the conditions are true - [FreeRTOS_ND.c#L953](https://github.com/FreeRTOS/FreeRTOS-Plus-TCP/blob/dev/IPv6_integration/source/FreeRTOS_ND.c#L953)

Reply:
Sure, just saw the check for `pxEndPoint->bits.bIPv6` in line 698. Marking it resolved. Thanks